cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -74,7 +74,7 @@ func main() {
 
 			c, err := ccmOptions.Config(app.ControllerNames(app.DefaultInitFuncConstructors), app.ControllersDisabledByDefault.List())
 			if err != nil {
-				fmt.Fprintf(os.Stderr, "%v\n", err)
+				_, _ = fmt.Fprintf(os.Stderr, "%v\n", err)
```

Comment:
wondering why we are making this change 🤔

Reply:
we are ignoring any error thrown by `Fprintf`, adding _ will make it more explicit

Reply:
Good question, could you add comment to the cod as well?

Reply:
sure!